corehq/apps/app_manager/templates/app_manager/partials/forms/case_config_ko_templates.html

```diff
@@ -247,7 +247,7 @@ <h4 class="panel-title panel-title-nolink">
           <p class="help-block"
              data-bind="html: validate,
                         visible: validate"></p>
-          <!-- ko if: $data.description && $parent.hasPrivilege -->
+          <!-- ko if: $data.description() && $parent.hasPrivilege() -->
```

Comment:
Nitpick: there's another `if: $data.description` below, though if description isn't there it'll just show an empty paragraph.

Reply:
I think if it's a solo observable you don't need to call it. it was `$data.description` before the change, with no `()`